package/src/skia/web/JsiSkPicture.ts

```diff
@@ -33,8 +35,8 @@ export class JsiSkPicture
         ckEnum(tmx),
         ckEnum(tmy),
         ckEnum(mode),
-        localMatrix ? toValue(localMatrix) : undefined,
-        tileRect ? toValue(tileRect) : undefined
+        localMatrix ? JsiSkMatrix.fromValue(localMatrix) : undefined,
+        tileRect ? JsiSkRect.fromValue(this.CanvasKit, tileRect) : undefined
```

Comment:
I see why this is done, and it makes sense - but it makes the polymorphism look at bit strange - would it be better if all of the `fromValue` calls passes CanvasKit as the first parameter?

Reply:
This was the original idea, but static methods cannot be overloaded in TypeScript.